src/main/java/org/tarantool/MsgPackLite.java

```diff
@@ -7,15 +7,59 @@
 import java.io.OutputStream;
 import java.lang.reflect.Array;
 import java.math.BigInteger;
-import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.Callable;
 
 /**
- * forked from https://bitbucket.org/sirbrialliance/msgpack-java-lite
+ * A simple binary (de-)serializer based on {@code msgpack-java-lite} implementation.
+ *
+ * <p>
+ * Supported types for serialization:
+ *
+ * <ul>
+ *     <li>null as MsgPack Nil value</li>
+ *     <li>
+ *         primitive types and their wrappers (byte, char, short, int, long, float, double, boolean) as
+ *         the corresponding MsgPack format values.
+ *     </li>
+ *     <li>{@link java.math.BigInteger}, especially to represent a MsgPack uint 64 value</li>
+ *     <li>{@link org.tarantool.Code} as an integer value</li>
+ *     <li>{@link java.lang.String}</li>
+ *     <li>java arrays and {@link java.util.List} as an MsgPack array format</li>
+ *     <li>{@link java.util.Map} as an MsgPack map format</li>
+ *     <li>
+ *         {@link java.util.concurrent.Callable} that can return any object of the types
```

Comment:
A few tests for Code and Callable would make the coverage 100% in fact (except erroneous situations). That would be good, I think.

Maybe also it would be worthful to add one invalid msgpack deserialization case.